### ISSUE TITLE
warnings in latest cljs version

### ISSUE THREAD

**Issue by thedavidmeister** [gfredericks/exact#1]
```
WARNING: No such namespace: com, could not locate com.cljs, com.cljc, or Closure namespace "" at line 172 /Users/davidmeister/.boot/cache/tmp/Users/davidmeister/estimate-work/teu/-efe1f0/index.html.out/com/gfredericks/exact/impl.cljs
WARNING: Use of undeclared Var com/gfredericks at line 172 /Users/davidmeister/.boot/cache/tmp/Users/davidmeister/estimate-work/teu/-efe1f0/index.html.out/com/gfredericks/exact/impl.cljs
```

from cljs version `1.9.473`

**Comment by gfredericks** [comment]
Thanks; I spent 15 minutes looking at this and it's pretty opaque, so will probably take me a bit to figure out.

This isn't causing any problems beyond the warnings, correct?

**Comment by thedavidmeister** [comment]
currently just warnings afaics, but i haven't thoroughly tested anything.

**Comment by gfredericks** [comment]
It looks like these are related to the use of the https://github.com/gfredericks/goog-integer fork of the `goog.math.Integer` class.

I was hoping I could just remove that and upgrading clojurescript would get the fixed version of google closure, but that's not the case. In fact maven doesn't seem to have any clojure-published releases past June 2016, so you couldn't even opt-in to the fixed version.

So I guess I'll have to figure out why clojurescript doesn't like the goog-integer library.

**Comment by gfredericks** [comment]
On the other hand, by just waiting a little while I can benefit from a new closure release.

I've pushed some changes that upgrade the clojurescript and closure dependencies to their latest versions, and remove the goog-integer dependency. I no longer see any warnings.

I have no idea what the root cause was, but hopefully this solves the problem on your end?

I'll have a new release out in a day or so. You can also try it before then via `lein install`.

**Comment by thedavidmeister** [comment]
if i see anything broken i'll come back to complain ;)

**Comment by gfredericks** [comment]
Released as `0.1.11`.
